firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/common/CrashlyticsBackgroundWorker.java

```diff
@@ -44,6 +44,7 @@
   // A thread local to keep track of which thread belongs to this executor.
   private ThreadLocal<Boolean> isExecutorThread = new ThreadLocal<>();
 
+  @SuppressWarnings("FutureReturnValueIgnored")
```

Comment:
Any chance we can scope this to line 51 instead of having it on the whole constructor?

Reply:
Is this better?

Reply:
Yes, thank you! :)